diff --git a/dist/aliases.d.ts b/dist/aliases.d.ts
index 820b05582946525ae446126ac3eda45cf2413ec6..bd4fb9db4055c92e188753cd583ecb74ba5faf05 100644
--- a/dist/aliases.d.ts
+++ b/dist/aliases.d.ts
@@ -4,6 +4,11 @@
  * This source code is licensed under the ISC license.
  * See the LICENSE file in the root directory of this source tree.
  */
+// Custom Aliases
+export { default as AdultIcon } from './icons/adult.svelte';
+export { default as ClockFading } from './icons/clock-fading.svelte';
+export { default as CloudAlert } from './icons/cloud-alert.svelte';
+// End of Custom Aliases
 export { default as AArrowDownIcon } from './icons/a-arrow-down.svelte';
 export { default as LucideAArrowDown } from './icons/a-arrow-down.svelte';
 export { default as AArrowUpIcon } from './icons/a-arrow-up.svelte';
diff --git a/dist/aliases.js b/dist/aliases.js
index 30a56e986b7bfc8b5ca26fd55f833404d55a8f76..c7a2006dab5d2a695d9763bd8bdf3cccbf52dcc1 100644
--- a/dist/aliases.js
+++ b/dist/aliases.js
@@ -4,6 +4,17 @@
  * This source code is licensed under the ISC license.
  * See the LICENSE file in the root directory of this source tree.
  */
+// Custom Aliases
+// Adult aliases
+export { default as AdultIcon } from './icons/adult.svelte';
+export { default as LucideAdultIcon } from './icons/adult.svelte';
+// ClockFading aliases
+export { default as ClockFadingIcon } from './icons/clock-fading.svelte';
+export { default as LucideClockFadingIcon } from './icons/clock-fading.svelte';
+// CloudAlert aliases
+export { default as CloudAlertIcon } from './icons/cloud-alert.svelte';
+export { default as LucideCloudAlertIcon } from './icons/cloud-alert.svelte';
+// End of Custom Aliases
 // AArrowDown aliases
 export { default as AArrowDownIcon } from './icons/a-arrow-down.svelte';
 export { default as LucideAArrowDown } from './icons/a-arrow-down.svelte';
diff --git a/dist/icons/adult.svelte b/dist/icons/adult.svelte
new file mode 100644
index 0000000000000000000000000000000000000000..a17fae948f3bc769b90502625d4778e6bd177d53
--- /dev/null
+++ b/dist/icons/adult.svelte
@@ -0,0 +1,23 @@
+<script>/**
+ * @license lucide-svelte v0.455.0 - ISC
+ *
+ * This source code is licensed under the ISC license.
+ * See the LICENSE file in the root directory of this source tree.
+ */
+import Icon from '../Icon.svelte';
+const iconNode = [["path", { "d": "M6.5 10 L9 8" }], ["path", { "d": "M9 8 L9 16" }], ["path", { "d": "M14 2.2a10 10 0 1 0 7.8 7.8" }], ["path", { "d": "M18 4h4" }], ["path", { "d": "M20 6V2" }], ["circle", { "cx": "15", "cy": "10", "r": "2" }], ["circle", { "cx": "15", "cy": "14", "r": "2" }]];
+/**
+ * @component @name Adult
+ * @description Lucide SVG icon component, renders SVG Element with children.
+ *
+ * @see https://lucide.dev/guide/packages/lucide-svelte - Documentation
+ *
+ * @param {Object} props - Lucide icons props and any valid SVG attribute
+ * @returns {FunctionalComponent} Svelte component
+ *
+ */
+</script>
+
+<Icon name="adult" {...$$props} iconNode={iconNode}>
+  <slot/>
+</Icon>
diff --git a/dist/icons/adult.svelte.d.ts b/dist/icons/adult.svelte.d.ts
new file mode 100644
index 0000000000000000000000000000000000000000..b674c168b475c8ee5cb6988eaa307c5bc864f25a
--- /dev/null
+++ b/dist/icons/adult.svelte.d.ts
@@ -0,0 +1,33 @@
+/**
+ * @license lucide-svelte v0.455.0 - ISC
+ *
+ * This source code is licensed under the ISC license.
+ * See the LICENSE file in the root directory of this source tree.
+ */
+import { SvelteComponentTyped } from "svelte";
+import type { IconProps } from '../types.js';
+declare const __propDef: {
+    props: IconProps;
+    events: {
+        [evt: string]: CustomEvent<any>;
+    };
+    slots: {
+        default: {};
+    };
+};
+export type AdultProps = typeof __propDef.props;
+export type AdultEvents = typeof __propDef.events;
+export type AdultSlots = typeof __propDef.slots;
+/**
+ * @component @name Adult
+ * @description Lucide SVG icon component, renders SVG Element with children.
+ *
+ * @see https://lucide.dev/guide/packages/lucide-svelte - Documentation
+ *
+ * @param {Object} props - Lucide icons props and any valid SVG attribute
+ * @returns {FunctionalComponent} Svelte component
+ *
+ */
+export default class Adult extends SvelteComponentTyped<AdultProps, AdultEvents, AdultSlots> {
+}
+export {};
diff --git a/dist/icons/clock-fading.svelte b/dist/icons/clock-fading.svelte
new file mode 100644
index 0000000000000000000000000000000000000000..a6ea9b73daf3dd13e53b22fb313cbdc1b6da5f32
--- /dev/null
+++ b/dist/icons/clock-fading.svelte
@@ -0,0 +1,23 @@
+<script>/**
+ * @license lucide-svelte v0.455.0 - ISC
+ *
+ * This source code is licensed under the ISC license.
+ * See the LICENSE file in the root directory of this source tree.
+ */
+import Icon from '../Icon.svelte';
+const iconNode = [["path", { d: "M12 2a10 10 0 0 1 7.38 16.75" }], ["path", { d: "M12 6v6l4 2" }], ["path", { d: "M2.5 8.875a10 10 0 0 0-.5 3" }], ["path", { d: "M2.83 16a10 10 0 0 0 2.43 3.4" }], ["path", { d: "M4.636 5.235a10 10 0 0 1 .891-.857" }], ["path", { d: "M8.644 21.42a10 10 0 0 0 7.631-.38" }]];
+/**
+ * @component @name ClockFading
+ * @description Lucide SVG icon component, renders SVG Element with children.
+ *
+ * @see https://lucide.dev/guide/packages/lucide-svelte - Documentation
+ *
+ * @param {Object} props - Lucide icons props and any valid SVG attribute
+ * @returns {FunctionalComponent} Svelte component
+ *
+ */
+</script>
+
+<Icon name="clock-fading" {...$$props} iconNode={iconNode}>
+  <slot/>
+</Icon>
diff --git a/dist/icons/clock-fading.ts b/dist/icons/clock-fading.ts
new file mode 100644
index 0000000000000000000000000000000000000000..b511d1c8f725e5d8ac2ab93dffb00bcc54f2f8d1
--- /dev/null
+++ b/dist/icons/clock-fading.ts
@@ -0,0 +1,33 @@
+/**
+ * @license lucide-svelte v0.455.0 - ISC
+ *
+ * This source code is licensed under the ISC license.
+ * See the LICENSE file in the root directory of this source tree.
+ */
+import { SvelteComponentTyped } from "svelte";
+import type { IconProps } from '../types.js';
+declare const __propDef: {
+    props: IconProps;
+    events: {
+        [evt: string]: CustomEvent<any>;
+    };
+    slots: {
+        default: {};
+    };
+};
+export type ClockFadingProps = typeof __propDef.props;
+export type ClockFadingEvents = typeof __propDef.events;
+export type ClockFadingSlots = typeof __propDef.slots;
+/**
+ * @component @name ClockFading
+ * @description Lucide SVG icon component, renders SVG Element with children.
+ *
+ * @see https://lucide.dev/guide/packages/lucide-svelte - Documentation
+ *
+ * @param {Object} props - Lucide icons props and any valid SVG attribute
+ * @returns {FunctionalComponent} Svelte component
+ *
+ */
+export default class ClockFading extends SvelteComponentTyped<ClockFadingProps, ClockFadingEvents, ClockFadingSlots> {
+}
+export {};
diff --git a/dist/icons/cloud-alert.d.ts b/dist/icons/cloud-alert.d.ts
new file mode 100644
index 0000000000000000000000000000000000000000..34451244e240fc6e7bb86754021338024d33e276
--- /dev/null
+++ b/dist/icons/cloud-alert.d.ts
@@ -0,0 +1,33 @@
+/**
+ * @license lucide-svelte v0.455.0 - ISC
+ *
+ * This source code is licensed under the ISC license.
+ * See the LICENSE file in the root directory of this source tree.
+ */
+import { SvelteComponentTyped } from "svelte";
+import type { IconProps } from '../types.js';
+declare const __propDef: {
+    props: IconProps;
+    events: {
+        [evt: string]: CustomEvent<any>;
+    };
+    slots: {
+        default: {};
+    };
+};
+export type CloudAlertProps = typeof __propDef.props;
+export type CloudAlertEvents = typeof __propDef.events;
+export type CloudAlertSlots = typeof __propDef.slots;
+/**
+ * @component @name CloudAlert
+ * @description Lucide SVG icon component, renders SVG Element with children.
+ *
+ * @see https://lucide.dev/guide/packages/lucide-svelte - Documentation
+ *
+ * @param {Object} props - Lucide icons props and any valid SVG attribute
+ * @returns {FunctionalComponent} Svelte component
+ *
+ */
+export default class CloudAlert extends SvelteComponentTyped<CloudAlertProps, CloudAlertEvents, CloudAlertSlots> {
+}
+export {};
diff --git a/dist/icons/cloud-alert.svelte b/dist/icons/cloud-alert.svelte
new file mode 100644
index 0000000000000000000000000000000000000000..0f90f9b2bee8515eadf029743b9f356cb0eaf8a7
--- /dev/null
+++ b/dist/icons/cloud-alert.svelte
@@ -0,0 +1,23 @@
+<script>/**
+ * @license lucide-svelte v0.455.0 - ISC
+ *
+ * This source code is licensed under the ISC license.
+ * See the LICENSE file in the root directory of this source tree.
+ */
+import Icon from '../Icon.svelte';
+const iconNode = [["path", { d: "M12 12v4" }],["path", { d: "M12 20h.01" }],["path", { d: "M17 18h.5a1 1 0 0 0 0-9h-1.79A7 7 0 1 0 7 17.708" }]]
+/**
+ * @component @name CloudAlert
+ * @description Lucide SVG icon component, renders SVG Element with children.
+ *
+ * @see https://lucide.dev/guide/packages/lucide-svelte - Documentation
+ *
+ * @param {Object} props - Lucide icons props and any valid SVG attribute
+ * @returns {FunctionalComponent} Svelte component
+ *
+ */
+</script>
+
+<Icon name="cloud-alert" {...$$props} iconNode={iconNode}>
+  <slot/>
+</Icon>
diff --git a/dist/icons/index.d.ts b/dist/icons/index.d.ts
index 0e74c325b84e09493607c558138cdfe7645de9bd..8b84096278710789940411a0b76b6cd887821a14 100644
--- a/dist/icons/index.d.ts
+++ b/dist/icons/index.d.ts
@@ -4,6 +4,11 @@
  * This source code is licensed under the ISC license.
  * See the LICENSE file in the root directory of this source tree.
  */
+// Custom Icons
+export { default as Adult } from './adult.svelte';
+export { default as ClockFading } from './clock-fading.svelte';
+export { default as CloudAlert } from './cloud-alert.svelte';
+// End of Custom Icons
 export { default as AArrowDown } from './a-arrow-down.svelte';
 export { default as AArrowUp } from './a-arrow-up.svelte';
 export { default as ALargeSmall } from './a-large-small.svelte';
diff --git a/dist/icons/index.js b/dist/icons/index.js
index 0e74c325b84e09493607c558138cdfe7645de9bd..8b84096278710789940411a0b76b6cd887821a14 100644
--- a/dist/icons/index.js
+++ b/dist/icons/index.js
@@ -4,6 +4,11 @@
  * This source code is licensed under the ISC license.
  * See the LICENSE file in the root directory of this source tree.
  */
+// Custom Icons
+export { default as Adult } from './adult.svelte';
+export { default as ClockFading } from './clock-fading.svelte';
+export { default as CloudAlert } from './cloud-alert.svelte';
+// End of Custom Icons
 export { default as AArrowDown } from './a-arrow-down.svelte';
 export { default as AArrowUp } from './a-arrow-up.svelte';
 export { default as ALargeSmall } from './a-large-small.svelte';
